src/main/java/edu/ucsb/cs156/happiercows/jobs/MilkTheCowsJob.java

```diff
@@ -34,23 +35,27 @@ public String formatDollars(double amount) {
 
     @Override
     public void accept(JobContext ctx) throws Exception {
-        ctx.log("Starting to milk the cows");
+        
+            ctx.log("Starting to milk the cows");
 
-        Iterable<Commons> allCommons = commonsRepository.findAll();
+            Iterable<Commons> allCommons = commonsRepository.findAll();
 
-        for (Commons commons : allCommons) {
-            String name = commons.getName();
-            double milkPrice = commons.getMilkPrice();
-            ctx.log("Milking cows for Commons: " + name + ", Milk Price: " + formatDollars(milkPrice));
+            for (Commons commons : allCommons) {
+                String name = commons.getName();
+                double milkPrice = commons.getMilkPrice();
+                ctx.log("Milking cows for Commons: " + name + ", Milk Price: " + formatDollars(milkPrice));
```

Comment:
This is where you should check:

```suggestion
                if (!commons.gameInProgress()) {
                    ctx.log("Commons " + name + " is not currently in progress, and cows were not milked.");
                } else { 
                   ctx.log("Milking cows for Commons: " + name + ", Milk Price: " + formatDollars(milkPrice));
                   // etc...  be sure to close this brace...
```